mkt/websites/migrations/0005_delete_websites.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+from mkt.constants.applications import DEVICE_TV
+
+
+def delete_websites(apps, schema_editor):
+    Website = apps.get_model("websites", "Website")
+    Website.objects.exclude(devices=[DEVICE_TV.id]).delete()
```

Comment:
Just to be clear : at the moment TV websites *only* have the TV device set, and no other ?